client/src/pages/SubmissionView.vue

```diff
@@ -0,0 +1,507 @@
+<template>
+  <div v-if="!submission" class="q-pa-lg">
+    {{ $t("loading") }}
+  </div>
+  <article v-else>
+    <q-layout
+      data-cy="submission_view_layout"
+      view="hHh lpR fFr"
+      container
+      style="min-height: calc(100vh - 70px)"
+    >
+      <q-header reveal elevated class="bg-grey-9 text-white">
+        <q-toolbar>
+          <q-btn
+            dense
+            aria-label="Back to Submission Details"
+            flat
+            round
+            icon="arrow_back_ios_new"
+            :to="{
+              name: 'submission_details',
+              params: { id: props.id },
+            }"
+          />
+          <q-toolbar-title>
+            {{ submission.title }}
+          </q-toolbar-title>
+
+          <q-btn
+            aria-label="Toggle Inline Comments"
+            dense
+            flat
+            round
+            icon="question_answer"
+            @click="toggleRightDrawer"
+          />
+        </q-toolbar>
+      </q-header>
```

Comment:
I would make this a component, SubmissionReviewToolbar (or header)

Reply:
This has been refactored as suggested.